### PR TITLE
fix amp-analytics dispose bug

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

File: extensions/amp-analytics/0.1/amp-analytics.js [hunk 1]
```diff
@@ -82,9 +82,9 @@ export class AmpAnalytics extends AMP.BaseElement {
     this.isSandbox_ = false;
 
     /**
-     * @private {Object<string, RequestHandler>} A map of request handler with requests
+     * @private {Object<!RequestHandler>} A map of request handler with requests
```
Comment (zhouyx) [inline extensions/amp-analytics/0.1/amp-analytics.js]: But `this.request_` is still object<string, RequestHandler>

Comment (erwinmombay) [reply extensions/amp-analytics/0.1/amp-analytics.js]: yes, this is the shorthard for a map, by default key is string

Comment (zhouyx) [reply extensions/amp-analytics/0.1/amp-analytics.js]: I see. Never know that : ) Thanks for explanation.

**Comment by nainar** [comment]
Pinging and assigning to @erwinmombay for next action.

**Comment by erwinmombay** [comment]
@nainar woops. will get it green and merge today

**Comment by mrjoro** [comment]
@erwinmombay are you still planning on merging this?

/cc @ampproject/wg-analytics

**Comment by mrjoro** [comment]
Closing this for now since there are no updates; please reopen if needed.

**Pull request closed without merging**